src/aks-preview/azcli_aks_live_test/scripts/start_container.sh

```diff
@@ -13,9 +13,13 @@ container_name=${1:-"azcli-aks-live-test-container"}
 [[ -z "${IMAGE_NAME}" ]] && (echo "IMAGE_NAME is empty"; exit 1)
 [[ -z "${IMAGE_TAG}" ]] && (echo "IMAGE_TAG is empty"; exit 1)
 
-# dir
-pwd
-ls -alh
+# remove container
```

Comment:
1ES-AKS-Elastic-DevInfra-Auto-TearDown-Pool

Reply:
Good point, changed in commit [#892574f](https://github.com/Azure/azure-cli-extensions/pull/4222/commits/892574f6480470cf40bb6dde56d49ebf6f3ab9e4).